docs/knt-backend-doc.yml

```diff
@@ -0,0 +1,363 @@
+          schema:
+            type: integer
+          required: true
+          description: Numeric ID of the user to target
+      security:
+        - ApiKeyAuth: []
+      responses:
+        '204':
+          description: No content
+        '401':
+          $ref: "#/components/responses/UnauthorizedError"
+        '407':
+          $ref: "#/components/responses/ForbidenError"
+  /admin/products/{productId}:
+    delete:
+      tags:
+        - "Product Admin"
+      description: 'Delete a product'
+      parameters: 
+        - in: path
+          name: productId
+          schema:
+            type: integer
+          required: true
+          description: Numeric ID of the product to target
+      security:
+        - ApiKeyAuth: []
+      responses:
+        '204':
+          description: No content
+        '401':
+          $ref: "#/components/responses/UnauthorizedError"
+        '407':
+          $ref: "#/components/responses/ForbidenError"
+    put:
+      tags:
+        - "Product Admin"
+      description: 'Edit product'
+      security:
+        - ApiKeyAuth: []
+      parameters: 
+        - in: path
+          name: productId
+          schema:
+            type: integer
+          required: true
+          description: Numeric ID of the product to target
+      responses:
+        '200':
+          description: OK
+        '401':
+          $ref: "#/components/responses/UnauthorizedError"
+        '407':
+          $ref: "#/components/responses/ForbidenError"
+  /admin/users/{userId}/money:
+    post:
+      tags:
+        - "User Admin"
+      description: 'Add money to a users account'
+      security:
+        - ApiKeyAuth: []
+      parameters: 
+        - in: path
+          name: userId
+          schema:
+            type: integer
+          required: true
+          description: Numeric ID of the user to target
+      requestBody:
+        content:
+          application/json:
+            schema:
+              type: integer
+              description: Amount in cents
+      responses:
+        '204':
+          description: OK
+        '401':
+          $ref: "#/components/responses/UnauthorizedError"
+        '407':
+          $ref: "#/components/responses/ForbidenError"
+  /admin/users:
+    get:
+      tags:
+        - "User Admin"
+      description: 'Get full users'
+      parameters: []
+      security:
+        - ApiKeyAuth: []
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/fulluser"
+        '401':
+          $ref: "#/components/responses/UnauthorizedError"
+        '407':
+          $ref: "#/components/responses/ForbidenError"
+components:
+  schemas:
+    user:
+      type: object
+      properties:
+        id:
+          type: integer
+          description: The user ID.
+        firstName:
+          type: string
+          description: The users first name.
```

Comment:
Description like "The user first name." for the firstName field seems unnecessary to me, it is very obvious, so it can be removed. For the money, though, is descriptive and thus helpful.

Reply:
Okay but the open api standart forces the description, so like you have to write something

Reply:
Yeah, the problem is just if you have stuff like userName - that's a username there is no point in description. I'll try to come up with something useful later, maybe like say specify the max length that should be expected or character type, all those kind of details.

Reply:
Please dont, I do not want to code that

Reply:
That's just an example, but did u not add any kind of constraints? We cannot possibly have some random gibberish for a firstName field, for instance.